### ISSUE TITLE
Mozilla Firefox on Pootle

### ISSUE THREAD

**Issue by dwaynebailey** [translate/pootle#2142]
Depends on:
- [x] translate/translate#115 .properties files should be able to merge accelerators
- [x] translate/translate#2045 In Mozilla pofilter unchanged should not fail for dialogsize strings
- [x] translate/translate#2179 Do proper escaping for Android DTD files
- [x] translate/translate#2190 moz2po DTD conversion causes duplicate msgctxt entry
- [x] translate/translate#2253 Improve speed on build_firefox.sh
- [x] translate/translate-project#2254 Update toolkit installation for mozilla.locamotion.org

This is a tracker bug for all bugs and features that should be implemented to improve the Firefox localisation process on Pootle.

Candidate bugs and features would include:
- Reducing errors
- Improving speed of processing
- Goals
- Removing bash scripts to Python

Basically anything that makes this process replicable, faster, easier and with less errors.
